### PR TITLE
Fix issues in Windows migration test

### PR DESCRIPTION
**Pull request by jingxu97**
Fix issues for Windows CSI migration test

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/run-k8s-windows-migration.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ set -o nounset
 set -o errexit
 
 export GCE_PD_VERBOSITY=9
-
+E2E_GOOGLE_APPLICATION_CREDENTIALS=/usr/local/google/home/jinxu/Work/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/deploy/credentials/cloud-sa.json
```
Comment (Jiawei0227) [inline test/run-k8s-windows-migration.sh:13]: Is this required?

Comment (jingxu97) [reply test/run-k8s-windows-migration.sh:13]: no, my mistake.

**Comment by Jiawei0227** [comment]
Can you add a release note label to NONE?

**Pull request merged**